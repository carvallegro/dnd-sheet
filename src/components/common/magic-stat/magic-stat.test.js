import React from 'react'
import { shallow } from 'enzyme'

import MagicStat from './index'

describe('<MagicStat />', () => {
  it('renders', () => {
    const wrapper = shallow(<MagicStat />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with props', () => {
    const wrapper = shallow(<MagicStat name="Test stat" value={10} />)
    expect(wrapper).toMatchSnapshot()
  })
})
