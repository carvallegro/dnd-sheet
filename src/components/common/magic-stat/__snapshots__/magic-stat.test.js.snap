// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MagicStat /> renders 1`] = `
<styled.div
  justifySelf="inherit"
>
  <styled.div>
    <styled.p />
  </styled.div>
  <styled.h3 />
</styled.div>
`;

exports[`<MagicStat /> renders with props 1`] = `
<styled.div
  justifySelf="inherit"
>
  <styled.div>
    <styled.p>
      10
    </styled.p>
  </styled.div>
  <styled.h3>
    Test stat
  </styled.h3>
</styled.div>
`;
